src/view/cli.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { Header, Title } from '../component/header';
+import Text from '../component/text';
+import { List, ListContent } from '../component/list';
+import { Button, BackButton } from '../component/button';
+import { color, font } from '../component/style';
+
+//
+// CLI View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    alignItems: 'flex-start',
+    paddingBottom: 30,
+    paddingLeft: 50,
+    paddingRight: 50,
+  },
+});
+
+const CLIView = ({ store, nav }) => (
+  <Background color={color.blackDark} style={styles.wrapper}>
+    <Header separator>
+      <BackButton onPress={() => nav.goSettings()} />
+      <Title title="CLI" />
+      <Button disabled onPress={() => {}} />
+    </Header>
+    <Background color={color.cliBackground}>
+      <ListContent>
+        <List
+          data={store.logs.slice()}
+          renderItem={text => <LogItem text={text} />}
+          scrollToEnd={true}
+        />
+      </ListContent>
```

Comment:
This renders the scrollbar inside of the view. If we remove the `ListContent` and just add padding to the `LogItem` components it should look better.

Reply:
I changed this back to use `ListContent` since it's more consistent with the way we did thing before.